zbsprimarystorage/setup.py

```
@@ -0,0 +1,26 @@
+from setuptools import setup, find_packages
+import sys, os
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import

(F401)